frontend/appflowy_flutter/lib/workspace/presentation/settings/widgets/settings_appearance_view.dart

```diff
@@ -417,31 +475,21 @@ class ThemeSettingDropDown extends StatefulWidget {
 class _ThemeSettingDropDownState extends State<ThemeSettingDropDown> {
   @override
   Widget build(BuildContext context) {
-    return Row(
-      children: [
-        Expanded(
-          child: FlowyText.medium(
-            widget.label,
-            overflow: TextOverflow.ellipsis,
-          ),
-        ),
-        AppFlowyPopover(
-          key: widget.popoverKey,
-          direction: PopoverDirection.bottomWithRightAligned,
-          popupBuilder: widget.popupBuilder,
-          constraints: const BoxConstraints(
-            minWidth: 80,
-            maxWidth: 160,
-            maxHeight: 400,
-          ),
-          onClose: widget.onClose,
-          child: FlowyTextButton(
-            widget.currentValue,
-            fontColor: Theme.of(context).colorScheme.onBackground,
-            fillColor: Colors.transparent,
-          ),
-        ),
-      ],
+    return AppFlowyPopover(
+      key: widget.popoverKey,
+      direction: PopoverDirection.bottomWithRightAligned,
+      popupBuilder: widget.popupBuilder,
+      constraints: const BoxConstraints(
+        minWidth: 80,
+        maxWidth: 160,
+        maxHeight: 400,
+      ),
+      onClose: widget.onClose,
+      child: FlowyTextButton(
+        widget.currentValue,
+        fontColor: Theme.of(context).colorScheme.onBackground,
+        fillColor: Colors.transparent,
+      ),
```

Comment:
I think this file is too big, so can you refactor this and separate these widgets into other files while you are at it?

Reply:
https://github.com/AppFlowy-IO/AppFlowy/pull/3137/commits/80efef0277467b5ecc3ddc34615c6853ee7b4604